src/networkremote/incomingdataparser.cpp

```diff
@@ -370,3 +380,57 @@ Song IncomingDataParser::CreateSongFromProtobuf(
 
   return song;
 }
+
+void IncomingDataParser::AppendFilesToPlaylist(const pb::remote::Message& msg) {
+  if (files_root_folder_.isEmpty()) {  // should never happen...
+    qLog(Warning) << "Remote root dir is not set although receiving "
+                     "APPEND_FILES request...";
+    return;
+  }
+  QDir rootDir(files_root_folder_);
```

Comment:
`clang-format` should have caught this but it hasn't, all non-static variable names should be in snake_case. If in doubt, check the Google C++ style guide.

Reply:
hum ok, I'll correct that... and have  a look at Google C++ style guide.
since when Google dictate a standard for C++ style (and I guess other languages...) when I learned, the trend was camelCase...